packages/runtime/test/src/interpreter/opcode.ts

```diff
@@ -44,3 +46,21 @@ describe("Teal Opcodes basic assertions", function () {
     );
   });
 });
+
+describe("parseToBigint", () => {
+  const op = new Op();
+
+  it("should parse to bigint", () => {
+    let res = op.parseToBigInt(["0"]);
+
+    assert.equal(res, 0n);
+
+    res = op.parseToBigInt(["0", "1", "0"]);
+
+    assert.equal(res, 2n);
+
+    res = op.parseToBigInt(["1", "0", "0", "0", "0"]);
+
+    assert.equal(res, 16n);
+  });
+});
```

Comment:
please also add test for `op.parseToBigInt(["1"]);`, `op.parseToBigInt(["0", "0"])`

Reply:
ok